README.md

```diff
@@ -15,18 +48,21 @@ Yoti SDK carries out for you steps 6, 7 ,8 and the profile decryption in step 9.
 
 Yoti also allows you to enable user details verification from your mobile app by means of the Android (TBA) and iOS (TBA) SDKs. In that scenario, your Yoti-enabled mobile app is playing both the role of the browser and the Yoti app. By the way, your back-end doesn't need to handle these cases in a significantly different way. You might just decide to handle the `User-Agent` header in order to provide different responses for web and mobile clients.
 
+## References
 
-## Enabling the SDK
-To import the Yoti SDK inside your project, you can use your favourite dependency management system.
-If you are using NPM, you need to add the following dependency:
+* [AES-256 symmetric encryption][]
+* [RSA pkcs asymmetric encryption][]
+* [Protocol buffers][]
+* [Base64 data][]
```

Comment:
Remove these as you have listed them all again with their links on the next 4 lines